### ISSUE TITLE
Intel OpenCL runtime causes Crashes

### ISSUE THREAD

**Issue by Xaymar** [Xaymar/OBS-AMD-Advanced-Media-Framework#372]
This has been an issue for a long time, and still is not fixed by either AMD or Intel. Given that in every single repro of this crash the crash location is identical, this is a bug that really needs to be worked around somehow.

# Crash log (always identical)
```
00000091AAEF7F40 00007FFB46040819 00000091AAEF8088 00007FFB461FCD00 0000000000000001 00007FFB54A71818 igdrclneo64.dll!0x7ffb46040819
00000091AAEF7FA0 00007FFB4602C4A7 00007FFB549B3277 0000000000000001 0000000000000001 00007FFB462DA018 igdrclneo64.dll!0x7ffb4602c4a7
00000091AAEF8030 00007FFB4607CEDA 00008CFDE919636D 0000000000000000 0000000000000000 0000000000000000 igdrclneo64.dll!0x7ffb4607ceda
00000091AAEF80D0 00007FFB4602D10E 000001DAB26E8240 0000000000000000 0000000000000000 0000000000000000 igdrclneo64.dll!0x7ffb4602d10e
00000091AAEF8130 00007FFB460341E9 000001DAB26E8260 00007FFB46033C80 000001DAB26E8240 00000091AAEF8280 igdrclneo64.dll!0x7ffb460341e9
00000091AAEF8180 00007FFB549B32B5 00007FFB87B16420 0000000000000000 0000000000000000 00000091AAEF8710 intelopencl64.dll!0x7ffb549b32b5
00000091AAEF8670 00007FFB5497C85B 00007FFB549748A9 00007FFB87B24F78 00007FFB54970000 00007FFB87B24F78 intelopencl64.dll!0x7ffb5497c85b
00000091AAEF86A0 00007FFB87B011BC 00007FFB00000000 0000000000000003 0000000000000000 000000000180200A opencl.dll!0x7ffb87b011bc
00000091AAEF8710 00007FFB87B0351B 0000000000000000 0000000000000000 0000000000000002 00007FFB00000000 opencl.dll!0x7ffb87b0351b
00000091AAEF8B80 00007FFB8EC0986F 0000000000000000 00000091AAEF8D20 00000091AAEF8C70 0000000000000000 ntdll.dll!0x7ffb8ec0986f
00000091AAEF8BC0 00007FFB8BC941BA 0000000000000001 0000000000000000 0000006400000065 00007FFB4EBE0470 kernelbase.dll!0x7ffb8bc941ba
00000091AAEF8BF0 00007FFB87B02E2C 00007FFB4EB06C84 000001DAB6F6D538 0000000000000000 00007FFB4EA9C444 opencl.dll!0x7ffb87b02e2c
00000091AAEF8C20 00007FFB4E9E8796 0000000000000002 0000000000000000 000001DAB6F6D538 0000000000000000 amfrt64.dll!0x7ffb4e9e8796
00000091AAEF95C0 00007FFB4E9C73B0 0000000000000002 0000000000000000 0000000000000002 0000000000000004 amfrt64.dll!0x7ffb4e9c73b0
00000091AAEF96B0 00007FFB4E9685EE 000001DA9896A1D0 000001DAB7282F10 000001DAA096A530 00007FFB4EB32840 amfrt64.dll!0x7ffb4e9685ee
00000091AAEF9B50 00007FFB551553DF 000000010000003C 000001DA9842D080 0000000000000000 000000000000003C enc-amf.dll!Plugin::AMD::Encoder::Start+0x5f
00000091AAEF9C30 00007FFB5516974D 0000000000000001 0000000000000001 0000000000000000 00007FFB56DB7169 enc-amf.dll!Plugin::Interface::H264Interface::H264Interface+0x64d
00000091AAEF9D90 00007FFB55169CDC 000001DAA04109E0 000001DAA0410BE0 000001DAB755D700 00007FFB56DCB5C7 enc-amf.dll!Plugin::Interface::H264Interface::create+0x3c
```

**Comment by Xaymar** [comment]
Pinging @jp9000 for possible solutions to this, as I'm kinda out of ideas other than moving the entire encoder into another process and adding on-gpu encoding (though I doubt that will fix it).

**Comment by Xaymar** [comment]
Will be "fixed" by #378.
